nicolas-martin-castellanos/Tarea-4/Calculadora/index.js

```diff
@@ -0,0 +1,113 @@
+var operadora;
+var operadorb;
+var operacion;
+
+function inicial (){
+    var resultado = document.getElementById("resultado");    
+    var clear = document.getElementById("clear");    
+    var borrar = document.getElementById("borrar");    
+    var dividir = document.getElementById("dividir");    
+    var multiplicar = document.getElementById("multiplicar");    
+    var sumar = document.getElementById("sumar");    
+    var restar = document.getElementById("restar");    
+    var igual = document.getElementById("igual");
+    
+    var cero = document.getElementById("cero");    
+    var uno = document.getElementById("uno");    
+    var dos = document.getElementById("dos");    
+    var tres = document.getElementById("tres");    
+    var cuatro = document.getElementById("cuatro");    
+    var cinco = document.getElementById("cinco");    
+    var seis = document.getElementById("seis");    
+    var siete = document.getElementById("siete");    
+    var ocho = document.getElementById("ocho");
+    var nueve = document.getElementById("nueve");
+    var punto = document.getElementById("punto");
+
+    cero.onclick=function(e){
```

Comment:
@NicolasMartin23 , find a way more efficient way to assing the onclick events, you're repeating a lot of code